app/assets/stylesheets/replies.scss

```diff
@@ -40,7 +40,7 @@
   padding: 5px 20px 10px 20px;
   z-index: 2;
   background-color: $bg_color_selectionpopup;
-  min-width: 200px;
+  min-width: 250px;
```

Comment:
Why was this changed?

Reply:
Oh! With the previous value and the new buttons, the UI sometimes collapsed weirdly. Let me find a screenshot quickly!

Reply:
200px minimum
![image](https://github.com/glowfic-constellation/glowfic/assets/577128/47834e7a-0781-4183-8e11-13843a878b3d)

250px minimum
![image](https://github.com/glowfic-constellation/glowfic/assets/577128/61116914-ebb9-403f-92b0-5a42f033926e)

(This way we always keep the NPC + Character buttons on the same line, rather than wrapping them)

I can go more aggressive on the minimum, down to 220px, which maintains that property and has a lower chance of messing with smaller UIs - that might be good?

![image](https://github.com/glowfic-constellation/glowfic/assets/577128/f0facd95-89ce-40fb-88e2-2e19d3f466dd)

Reply:
Do you think we should drop this to 220px, or is 250px ok?